test/test_cuda.py

```diff
@@ -3947,6 +3947,20 @@ def test_nvml_get_handler(self):
     def test_temperature(self):
         self.assertTrue(0 <= torch.cuda.temperature() <= 150)
 
+    @unittest.skipIf(TEST_PYNVML, "pynvml/amdsmi is not available")
+    def test_memory_usage_in_bytes(self):
+        """
+        Verify memory usage in bytes
+        """
+        torch.cuda.empty_cache()
+        a = torch.cuda.memory_usage_in_bytes()
+        num_bytes = 256 * 1024**2
+        _ = torch.zeros(num_bytes, dtype=torch.int8, device="cuda")
+        torch.cuda.synchronize()
+        b = torch.cuda.memory_usage_in_bytes()
+        mem_bytes = b - a
+        self.assertTrue(mem_bytes > num_bytes // 2, mem_bytes < num_bytes * 8)
```

Comment:
What's the `//2` and `* 8` for? Are they fudge factors?

Reply:
probably due to your above comment on memory overhead, in my testing the mem_bytes can be larger than the tensor size especially for small tensor sizes. Hence I added this range as in this test we are primarily interested in the order of the mem, not exact value.